autosubmit/autosubmit.py

```diff
@@ -2396,6 +2404,12 @@ def run_experiment(expid, notransitive=False, start_time=None, start_after=None,
                         exp_history.finish_current_experiment_run()
                     except Exception:
                         Log.warning("Database is locked")
+                # Create rocrate object if requested
```

Comment:
I think we can even drop this comment if you'd like. Your code is so simple that others should be able to understand it well :slightly_smiling_face: https://dook.pro/blog/programming/21-keep-it-clean-coding-good-practices-comments